docs/pages/docs/handbook/linting/eslint.mdx

```diff
@@ -0,0 +1,124 @@
+import { Tabs, Tab } from "../../../../components/Tabs";
+
+# ESLint in a monorepo
+
+## Sharing config
+
+Sharing an ESLint config across workspaces can be a boon to productivity by making all your workspaces more consistent.
+
+Let's imagine a monorepo like this:
+
+```
+apps
+├─ docs
+│  ├─ package.json
+│  ├─ .eslintrc.js
+├─ web
+│  ├─ package.json
+│  ├─ .eslintrc.js
+packages
+├─ eslint-config-custom
+│  ├─ index.js
+│  ├─ package.json
+```
+
+We've got a package called `eslint-config-custom`, and two applications, each with their own `.eslintrc.js`.
+
+### Our `eslint-config-custom` package
+
+Our `eslint-config-custom` file contains only a single file, `index.js`. It looks like this.
+
+```js filename="packages/eslint-config-custom/index.js"
+module.exports = {
+  extends: ["next", "turbo", "prettier"],
+  rules: {
+    "@next/next/no-html-link-for-pages": "off",
+    "react/jsx-key": "off",
+  },
+};
```

Comment:
I don't know how, but it would be nice to combine some of these code samples with what's in `examples/` directory. These snippets are hard to maintain and keep up to date over time